x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -43,15 +46,26 @@ export const ComplianceDashboard = () => {
       <div
         data-test-subj={DASHBOARD_CONTAINER}
         css={css`
-          max-width: 1600px;
+          max-width: 1440px;
           margin-left: auto;
           margin-right: auto;
         `}
       >
-        <SummarySection complianceData={getDashboardData.data!} />
-        <EuiSpacer />
-        <BenchmarksSection complianceData={getDashboardData.data!} />
-        <EuiSpacer />
+        {INTERNAL_FEATURE_FLAGS.showNewDashboard ? (
+          <>
+            <CloudSummarySection complianceData={getDashboardData.data!} />
```

Comment:
sidenote: we should find a way to not use `data!` because we know it's ok in this particular place.

Reply:
i agree but i dont want to hold this pr in review stage because of that, i would preferably like to solve that in typescript and not by using an if statement, so I'm not sure whats the optimal solution but lets work on it when its time